Sources/WordPressKit/Services/StatsServiceRemoteV2.swift

```diff
@@ -342,6 +372,28 @@ public extension StatsServiceRemoteV2 {
     }
 }
 
+// MARK: - Email Opens
+
+public extension StatsServiceRemoteV2 {
+    func getEmailOpens(for postID: Int, completion: @escaping ((StatsEmailOpensData?, Error?) -> Void)) {
```

Comment:
What do you think about using async functions instead of calls for the new API calls? I presume the callers are written in Swift too, which may prefer async functions?